### ISSUE TITLE
[0.6.2] Vue language server crashes extremely often

### ISSUE THREAD

**Issue by icehaunter** [octref/vetur#156]
This is the error log:
```
[Info  - 5:34:00 PM] Connection to server got closed. Server will restart.
/home/icehaunter/.vscode/extensions/octref.vetur-0.6.2/client/server/node_modules/typescript/lib/typescript.js:23318
                var symbol = container.locals.get(node.text);
                                             ^

TypeError: Cannot read property 'get' of undefined
    at isNameOfExportsOrModuleExportsAliasDeclaration (/home/icehaunter/.vscode/extensions/octref.vetur-0.6.2/client/server/node_modules/typescript/lib/typescript.js:23318:46)
    at bindStaticPropertyAssignment (/home/icehaunter/.vscode/extensions/octref.vetur-0.6.2/client/server/node_modules/typescript/lib/typescript.js:23396:17)
    at bindWorker (/home/icehaunter/.vscode/extensions/octref.vetur-0.6.2/client/server/node_modules/typescript/lib/typescript.js:23079:29)
    at bind (/home/icehaunter/.vscode/extensions/octref.vetur-0.6.2/client/server/node_modules/typescript/lib/typescript.js:22974:13)
    at visitNode (/home/icehaunter/.vscode/extensions/octref.vetur-0.6.2/client/server/node_modules/typescript/lib/typescript.js:14662:20)
    at Object.forEachChild (/home/icehaunter/.vscode/extensions/octref.vetur-0.6.2/client/server/node_modules/typescript/lib/typescript.js:14826:24)
    at bindEachChild (/home/icehaunter/.vscode/extensions/octref.vetur-0.6.2/client/server/node_modules/typescript/lib/typescript.js:21771:16)
    at bindBinaryExpressionFlow (/home/icehaunter/.vscode/extensions/octref.vetur-0.6.2/client/server/node_modules/typescript/lib/typescript.js:22416:17)
    at bindChildrenWorker (/home/icehaunter/.vscode/extensions/octref.vetur-0.6.2/client/server/node_modules/typescript/lib/typescript.js:21831:21)
    at bindChildren (/home/icehaunter/.vscode/extensions/octref.vetur-0.6.2/client/server/node_modules/typescript/lib/typescript.js:21746:17)
```

**Comment by sebastianmacias** [comment]
Same here, keep having to go back to vetur 0.5.6

**Comment by octref** [comment]
Dup of #131. Should be fixed in 0.6.4.
